apps/site/src/app/(main)/portal/@applicant/ApplicantPortal.tsx

```diff
@@ -34,6 +34,7 @@ async function Portal() {
 			<h2 className="text-4xl font-semibold">Status</h2>
 			<VerticalTimeline status={status as PortalStatus} />
 			<Message status={status as PortalStatus} />
+			{isAccepted && <hr />}
 			{isAccepted && <SignWaiver status={status as PortalStatus} />}
```

Comment:
Suggestion: Maybe combine these two if they have the same conditional?

Reply:
I'll need to reconsider based on the other conditional introduced in #252.